R/mergeRaster.R

```diff
@@ -109,5 +109,9 @@ setMethod(
       }
       x[[i]] <- crop(r, extent(xminCut, xmaxCut, yminCut, ymaxCut))
     }
-    return(do.call(raster::merge, x))
+    y <- do.call(raster::merge, x)
+    yname <- gsub("_tile", "", names(x[[1]])) %>%
+      gsub("\\d", "", .)
```

Comment:
yes. you are right. it should remove all the numbers after "_tile". I will
correct this bug.

On Thu, Jun 16, 2016 at 1:37 PM, Alex Chubaty <notifications@github.com>
wrote:

> In R/mergeRaster.R
> <https://github.com/PredictiveEcology/SpaDES/pull/286#discussion_r67419546>
> :
>
> > @@ -109,5 +109,9 @@ setMethod(
> >        }
> >        x[[i]] <- crop(r, extent(xminCut, xmaxCut, yminCut, ymaxCut))
> >      }
> > -    return(do.call(raster::merge, x))
> > +    y <- do.call(raster::merge, x)
> > +    yname <- gsub("_tile", "", names(x[[1]])) %>%
> > +      gsub("\\d", "", .)
>
> removes all numbers from the name, which isn't what we want. some rasters
> will have numbers in their names before splitting
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PredictiveEcology/SpaDES/pull/286/files/a733cb411ca2c41632018cd6f2372a7bd13146e0#r67419546>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ-1fIyuOzMNp7zMvIlL7gtcDpmk5IJ9ks5qMbQngaJpZM4I3FuJ>
> .
>



-- 
Yong Luo
Postdoctoral Researcher

Canadian Forest Service
Natural Resources Canada
Pacific Forestry Centre
506 Burnside Road W
Victoria, BC  V8Z 1M5
Canada
phone: +1-250-298-2566
&
Faculty of Forestry
University of British Columbia
2424 Main Mall
Vancouver, BC V6T 1Z4
Canada